device/arista/x86_64-arista_7280cr3mk_32p4/Arista-7280CR3-C40/gearbox_100G_PAM4.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0"?>
+<root>
+    <name>CSDK-B52</name>
+    <phy_addr>0</phy_addr>
+    <mode>gearbox</mode>
+    <topology>2</topology>
+    <tx-taps>
+        <PAM4>2,-8,17,0,0</PAM4>
+        <NRZ>0,-8,17,0,0</NRZ>
+    </tx-taps>
+    <tx-taps-scale>0,0,1,0,0</tx-taps-scale>
+
+    <lane id="0" system-side="true" />
+    <lane id="1" system-side="true" />
```

Comment:
why do we need this file? we have phy_config_1.json, then we should not need this xml? it seems redundant information here.

Reply:
This file is used by credo driver directly.  I'll check whether some items in it can be removed. Still, there are some have to be kept for the current driver design, like <name>CSDK-B52</name> for determining the chip model to initialize.

Reply:
It turns out that for now the items in this file cannot be removed, as they are still used by the driver.

Reply:
we will keep it for now, can you check with credo to remove the support for this file?

Reply:
sure